service/entityresolution/entity_resolution.proto

```diff
@@ -62,11 +62,61 @@ message EntityNotFoundError {
 }
 
 
+/*
+   Example: Get Entity chains for tokens aaaaaa and bbbbbb
+   {
+     "tokens": [
+       "aaaaaaa",
+       "bbbbbbbb"
+     ]
+   }
+
+*/
+message GetEntityChainFromJwtRequest {
+    repeated authorization.Token tokens = 1;
+}
+
+/*
+   Example: Return the entity chains from the provided tokens
+   {
+     "entity_chains": [
+    {
+      "id": "tok1",
+      "entities": [
+        {
+          "clientId": "client1"
+        }
+      ]
+    },
+    {
+      "id": "tok2",
+      "entities": [
+        {
+          "userName": "alice",
+          "clientId": "client2"
+        }
+      ]
+    }
+  ]
+   }
+
+*/
+message GetEntityChainFromJwtResponse {
+  repeated authorization.EntityChain entity_chains = 1;
+}
+
+
 service EntityResolutionService {
   rpc ResolveEntities(ResolveEntitiesRequest) returns (ResolveEntitiesResponse) {
     option (google.api.http) = {
         post: "/entityresolution/resolve"
         body: "*";
     };
   }
+  rpc GetEntityChainFromJwt(GetEntityChainFromJwtRequest) returns (GetEntityChainFromJwtResponse) {
```

Comment:
This is probably a Nit, but it would likely keep the Casbin route authz policy cleaner and easier to read if we call the RPCs `CreateEntityChainFromJwt` or `BuildEntityChainFromJwt` instead of `Get...` since this is a POST http verb and not a GET?

Reply:
I don't think this is a blocker regardless because this route is internal to the platform (for now) so it can be changed later.

Reply:
i agree i like CreateEntityChainFromJwt better, ill update it